speculos/main.py

```diff
@@ -170,13 +190,12 @@ def run_qemu(s1: socket.socket, s2: socket.socket, args: argparse.Namespace) ->
     else:
         logger.warn(f"Cx lib {cxlib_filepath} not found")
 
-    if args.model == "stax":
-        fonts_filepath = f"/fonts/{args.model}-fonts-{args.apiLevel}.bin"
-        fonts = pkg_resources.resource_filename(__name__, fonts_filepath)
-        if os.path.exists(fonts):
-            argv += ['-f', fonts]
-        else:
-            logger.warn(f"Fonts {fonts_filepath} not found")
+    fonts_filepath = f"/fonts/{args.model}-fonts-{args.apiLevel}.bin"
+    fonts = pkg_resources.resource_filename(__name__, fonts_filepath)
+    if os.path.exists(fonts):
+        argv += ['-f', fonts]
+    elif not use_bagl:
+        logger.warn(f"Fonts {fonts_filepath} not found")
```

Comment:
```suggestion
    if not use_bagl:
        if os.path.exists(fonts):
            argv += ['-f', fonts]
        else:
            logger.warn(f"Fonts {fonts_filepath} not found")
```
Cause fonts are only needed if bagl is not used? And shouldn't be provided if the file exist but the app use bagl.

Reply:
It doesn't mind but yes, fonts file is useless for BAGL